index.html

```diff
@@ -1,29 +1,230 @@
+          <div class="work-text2">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </div>
+          <ul class="new-lang">
+            <button class="last-lang"><li>HTML</li></button>
+            <button class="last-lang"><li>Bootstrap</li></button>
+            <button class="last-lang"><li>Ruby</li></button>
+          </ul>
+        </li>
+      </ul>
+      <div class="background">
+        <img src="icons/background.png" class="background-img" />
+        <div class="back-button">
+          <button class="button1">See Project</button>
+        </div>
+      </div>
+      <ul class="container">
+        <li class="grid-container">
+          <div class="work-text1">Profesional Art Printing Data</div>
+          <div class="work-text2">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </div>
+          <ul class="new-lang">
+            <button class="last-lang"><li>HTML</li></button>
+            <button class="last-lang"><li>Bootstrap</li></button>
+            <button class="last-lang"><li>Ruby</li></button>
+          </ul>
+        </li>
+      </ul>
+      <div class="background">
+        <img src="icons/background.png" class="background-img" />
+        <div class="back-button">
+          <button class="button1">See Project</button>
+        </div>
+      </div>
+      <ul class="container">
+        <li class="grid-container">
+          <div class="work-text1">Profesional Art Printing Data</div>
+          <div class="work-text2">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </div>
+          <ul class="new-lang">
+            <button class="last-lang"><li>HTML</li></button>
+            <button class="last-lang"><li>Bootstrap</li></button>
+            <button class="last-lang"><li>Ruby</li></button>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="about">
+      <h3 class="about-me">About me</h3>
+      <div class="about-text">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </div>
+      <ul>
+        <li><button class="resume">Get my resume</button></li>
+      </ul>
+      <div class="new-vector">
+        <img src="icons/new-vector.png" />
+      </div>
+
+      <div class="language">
+        <ul>
+          <li class="tech-titles">
+            <img src="icons/rect1.png" alt="react1" />
+            Language
+          </li>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
```

Comment:
- [ ] I can see you implemented some of the requested changes by the previous reviewer but there is  something you are still missing out kindly consider warping the technologies (e.g., "JavasScript", "HTML" and "CSS") in a ul as  As a best practice, it's always a good idea to combine a lit of items that are related in a ul and li tags and it will also help you to meet the below requirement 
> The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`